WordPress/Classes/ViewRelated/NUX/LoginEmailViewController.swift

```diff
@@ -135,15 +135,18 @@ class LoginEmailViewController: LoginViewController, SigninKeyboardResponder {
             return
         }
 
-        let button = UIButton.googleLoginButton()
+        let button = WPStyleGuide.googleLoginButton()
         let buttonWrapper = UIView()
         buttonWrapper.addSubview(button)
         stackView.addArrangedSubview(buttonWrapper)
         button.addTarget(self, action: #selector(googleLoginTapped), for: .touchUpInside)
 
+        // Ensure height.
+        button.heightAnchor.constraint(greaterThanOrEqualToConstant: 40.0).isActive = true
```

Comment:
The `40.0` should use a value from a `Constants` enum. `labelMinHeight` from the original version did this (but on the label, not the button) and is now unused. You could move this into `googleLoginButton` and use that enum? Also, I think in the future we'll reuse that button code as there are some other multiline buttons in login that need fixing.